components/UserButton.vue

```diff
@@ -2,13 +2,12 @@
   <v-menu offset-y>
     <template #activator="{ on, attrs }">
       <div class="d-flex align-center" v-bind="attrs" v-on="on">
-        <v-avatar size="48" color="primary" class="mr-4">
-          <v-img
-            v-if="userImage"
-            :src="userImage"
-            :alt="$auth.user.Name"
-          ></v-img>
-          <v-icon v-else dark>mdi-account</v-icon>
+        <v-avatar :size="avatarSize" color="primary" class="mr-4">
+          <v-img :src="userImage" :alt="$auth.user.Name">
+            <template #placeholder>
+              <v-icon dark>mdi-account</v-icon>
```

Comment:
Can you confirm that changing from between light/dark theme keeps the icon the same color (white)?

Reply:
Yup, the `dark` prop is static and not linked to the `$vuetify.theme.dark` property I've noticed. Here are the pics:
![print_screen_2020-12-14-12-02-46](https://user-images.githubusercontent.com/1619359/102073971-5e6e2480-3e04-11eb-9215-e6df8c966449.png)
![print_screen_2020-12-14-12-02-41](https://user-images.githubusercontent.com/1619359/102073997-63cb6f00-3e04-11eb-9e88-b0711aa33049.png)

Reply:
Cool Thanks for checking